community-edition/readme.md

```diff
@@ -1,48 +1,57 @@
+# Introduction
+
+Community Edition is a free tool designed to help developers deploy the full Hubs stack on cloud computing software of their choosing. Community Edition simplifies and automates most of the complex deployment process using Kubernetes, a containerized software orchestration system.
+
+Community Edition is designed for developers capable of working with the full Hubs stack and of navigating complex application infrastructure. With Community Edition, developers are responsible for designing, building, hosting, and maintaining their software throughout its lifetime. In order to create a production-ready version of Hubs similar to Hubs Cloud, Developers using Community Edition will need to implement additional features and customizations outside of those listed in this repo. See "Considerations for Production Environment".
+
+# Why Kubernetes
+
+Kubernetes is an industry standard for allowing developers to build, deploy, and scale application efficiently and reliably. Benefits to Kubernetes include:
+
+- [Portability, Extensibility, and Open Source](https://kubernetes.io/docs/concepts/overview/)
+- [Availability in many cloud environments](https://kubernetes.io/docs/setup/production-environment/turnkey-solutions/)
+- Many options for single server deployments (ie. minikube, k3s, microk8s, kind)
+
+# Prerequisites
+
+Before deploying the charts in this repo, you will need to choose and configure the following services...
```

Comment:
a few issues in this sentence -- they are not all services, managed k8s is but local k8s is just a software you install, opening up the ports is also not a service